internal/view/app.go

```diff
@@ -173,6 +172,49 @@ func (a *App) suggestCommand() model.SuggestionFunc {
 	}
 }
 
+func (a *App) suggestSubCommand(command string) []string {
+	cmds := strings.Split(command, " ")
+	if len(cmds[0]) == 0 || len(cmds) != 2 {
```

Comment:
I have several suggestions here:
1. I think it might be a good idea to use strings.Fields instead to pick up potential typo ie `po   fred`
2. Best to make this call a helper and take in dependencies aka context, ns maps. So we don't have to refetch as there could be lots there. I think we should provide apis ie ContextNames and NamespaceNames which could return a map of names??
3. Would be great to put some tests on this call.

Reply:
> I have several suggestions here:
> 
> 1. I think it might be a good idea to use strings.Fields instead to pick up potential typo ie `po   fred`
> 2. Best to make this call a helper and take in dependencies aka context, ns maps. So we don't have to refetch as there could be lots there. I think we should provide apis ie ContextNames and NamespaceNames which could return a map of names??
> 3. Would be great to put some tests on this call.

1. done
2. I'm trying to understand this suggestion, do you mean implementing it as a method in a struct while caching the contexts and the namespaces in the struct?
3. I'll provide test cases later